src/modules/UI/scenes/TransactionList/style.js

```diff
@@ -102,9 +121,9 @@ export const styles = {
     justifyContent: 'flex-start',
     backgroundColor: THEME.COLORS.TRANSPARENT
   },
-  currentBalanceBoxBitssWrap: {
+  currentBalanceBoxBitsWrap: {
```

Comment:
what is this?

Reply:
It's an adjustment to the styling and also a spelling fix